crates/utils/src/utils/slurs.rs

```diff
@@ -30,24 +30,27 @@ pub(crate) fn slur_check<'a>(
   }
 }
 
-pub fn build_slur_regex(regex_str: Option<&str>) -> Option<Regex> {
+pub fn build_slur_regex(regex_str: Option<&str>) -> Option<LemmyResult<Regex>> {
```

Comment:
Another big change, was that I turned all these from `Option<Regex>`, to an `Option<Result<Regex>>`. This was necessary to avoid expects / runtime panics.